lib/managers/enterprise.js

```diff
@@ -0,0 +1,165 @@
+ */
+Enterprise.prototype.userStatuses = Object.freeze({
+	ACTIVE: 'active',
+	INACTIVE: 'inactive',
+	CANNOT_DELETE_OR_EDIT: 'cannot_delete_edit',
+	CANNOT_DELETE_EDIT_OR_UPLOAD: 'cannot_delete_edit_upload'
+});
+
+/**
+ * List of valid roles
+ * @readonly
+ * @enum {EnterpriseRole}
+ */
+Enterprise.prototype.userRoles = Object.freeze({
+	USER: 'user',
+	COADMIN: 'coadmin'
+});
+
+/**
+ * Get a list of users in the current enterprise
+ *
+ * API Endpoint: '/users'
+ * Method: GET
+ *
+ * @param {?Object} options - Optional parameters, can be left null in most cases
+ * @param {string} [options.filter_term] - Filter the results to only users starting with the filter_term in either the name or the login
+ * @param {int} [options.limit=100] - The number of records to return
+ * @param {int} [options.offset=0] - The record at which to start
+ * @param {EnterpriseUserType} [options.user_type=managed] - The type of user to search for
+ * @param {Function} callback - Passed the list of users if successful, error otherwise
+ * @returns {void}
+ */
+Enterprise.prototype.getUsers = function(options, callback) {
+
+	var apiPath = urlPath(USERS_PATH),
+		params = {
+			qs: options
+		};
+
+	this.client.get(apiPath, params, this.client.defaultResponseHandler(callback));
+};
+
+/**
+ * Invites a user to join the enterprise
+ *
+ * API Endpoint: '/invites'
+ * Method: POST
+ *
+ * @param {string} enterpriseID - The ID of the enterprise to invite the user to
+ * @param {string} email - The email address of the user to invite
+ * @param {Function} callback - Passed the invite object if successful, error otherwise
+ * @returns {void}
+ */
+Enterprise.prototype.inviteUser = function(enterpriseID, email, callback) {
+
+	var apiPath = urlPath(INVITES_PATH),
+		params = {
+			body: {
+				enterprise: {
+					id: enterpriseID
+				},
+				actionable_by: {
+					login: email
+				}
+			}
+		};
+
+	this.client.post(apiPath, params, this.client.defaultResponseHandler(callback));
+};
+
+/**
+ * Create a new user in the current enterprise
+ *
+ * API Endpoint: '/users'
+ * Method: POST
+ *
+ * @param {string} login - The email address this user uses to login
+ * @param {string} name - The name of this user
+ * @param {?Object} options - Optional parameters, can be left null in most cases
+ * @param {EnterpriseRole} [options.role] - This user’s enterprise role
+ * @param {string} [options.language] - The user's language
+ * @param {bool} [options.is_sync_enabled] - Whether or not this user can use Box Sync
+ * @param {string} [options.job_title] - The user’s job title
+ * @param {string} [options.phone] - The user’s phone number
+ * @param {string} [options.address] - The user’s address
+ * @param {int} [options.space_amount] - The user’s total available storage space in bytes
+ * @param {Array} [options.tracking_codes] - An array of key/value pairs set by the user’s admin
+ * @param {EnterpriseUserStatus} [options.status] - The user's status
```

Comment:
The docs don't list "status" as a param (https://docs.box.com/reference#create-an-enterprise-user), though it does come back in the response. Can it be set via API?